Tests/SonarScanner.MSBuild.Tasks.IntegrationTests/app.config

```diff
@@ -34,7 +34,7 @@
 
         <assemblyIdentity name="System.Collections.Immutable" publicKeyToken="b03f5f7f11d50a3a" culture="neutral" />
 
-        <bindingRedirect oldVersion="0.0.0.0-1.2.5.0" newVersion="1.2.5.0" />
+        <bindingRedirect oldVersion="0.0.0.0-5.0.0.0" newVersion="5.0.0.0" />
```

Comment:
Again, these updates were applied by VS.

I tried manually deleted the binding redirects for MSBuild earlier in the file, since the tests all passed locally without them. However, other tests failed on the build agent so I re-instated them.